skills/ghds/SKILL.md

```
@@ -0,0 +1,113 @@
+// Token CSS import — once, at app entry
+import '@ghds/tokens/css';
+```
+
+```tsx
+// FormField composition — wraps exactly one control, wires label/error/aria
+<FormField label="Email" helperText="We'll never share this." error={emailError}>
+  <Input type="email" />
+</FormField>
+```
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)